client/web/web.go

```diff
@@ -65,13 +65,32 @@ type Server struct {
 	browserSessions sync.Map
 }
 
+// ServerType specifies the type of a web.Server.
+// If "tailscaled", a full management client is being served.
```

Comment:
I suspect we are going have other enforcement for DNS rebinding that is going to make it such that the full management mode will only ever work over the tailscale interface, even if you were to try and enable it in other environments.  That's certainly the only way it's ever intended to be used, so might be worth adding to the docs here.

Also, maybe move these per-value docs off of the type and onto the vars below?